kumascript/macros/HTTPSidebar.ejs

```diff
@@ -244,7 +244,7 @@ var text = mdn.localStringMap({
     'Evolution': 'HTTP 的发展',
     'ResourcesURI': '资源和 URI',
     'Identifying': '标识互联网上的内容',
-    'DataURIs': 'Data URI',
+    'DataURLs': 'Data URL',
```

Comment:
LGTM. Thanks @wbamberg :+1:

Reply:
Thank you @yin1999 ! I should perhaps have asked the l10n folk but I thought this was a safe change 😓 .